<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="file_view_2">
       <attribute name="title">
        <string>基础视图</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item row="0" column="2">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="0">
             <widget class="QListWidget" name="listWidgetLabel"/>
            </item>
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidgetRelation">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="rowCount">
               <number>0</number>
              </property>
              <property name="columnCount">
               <number>0</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>标签</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidgetAttr">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="rowCount">
               <number>0</number>
              </property>
              <property name="columnCount">
               <number>0</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QListWidget" name="listWidgetKw">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>领域信息</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>特征信息</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QTreeWidget" name="treeWidgetField">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QComboBox" name="comboBoxTreeViewType">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>文件类型</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>文件领域</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonReload">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeView" name="treeView">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项(&amp;O)</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionArrangeInfo"/>
    <addaction name="actionLabelMaintain"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>搜索(&amp;S)</string>
    </property>
    <addaction name="actionQuickSearch"/>
    <addaction name="actionAdvancedSearch"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_S"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefrashFiles"/>
   <addaction name="actionIndex"/>
   <addaction name="actionBuildRelation"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>设置(&amp;S)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>标签词云界面(&amp;L)</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/index.png</normaloff>:/images/icons/index.png</iconset>
   </property>
   <property name="text">
    <string>更新索引</string>
   </property>
   <property name="toolTip">
    <string>更新文件索引</string>
   </property>
  </action>
  <action name="actionBuildRelation">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/relation.png</normaloff>:/images/icons/relation.png</iconset>
   </property>
   <property name="text">
    <string>关系重建</string>
   </property>
   <property name="toolTip">
    <string>关系重建</string>
   </property>
  </action>
  <action name="actionAdvancedSearch">
   <property name="text">
    <string>高级搜索(&amp;A)</string>
   </property>
  </action>
  <action name="actionQuickSearch">
   <property name="text">
    <string>快速搜索(&amp;Q)</string>
   </property>
  </action>
  <action name="actionRefrashFiles">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/refresh.png</normaloff>:/images/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>更新文件库</string>
   </property>
   <property name="toolTip">
    <string>更新文件库</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>在文件夹中显示(&amp;F)</string>
   </property>
  </action>
  <action name="actionArrange">
   <property name="text">
    <string>整理此分类到(&amp;S)...</string>
   </property>
  </action>
  <action name="actionArrangeInfo">
   <property name="text">
    <string>文件整理(&amp;R)</string>
   </property>
  </action>
  <action name="actionFieldFile">
   <property name="text">
    <string>查看领域下文件(&amp;C)</string>
   </property>
  </action>
  <action name="actionImagine">
   <property name="text">
    <string>文件联想(&amp;I)</string>
   </property>
  </action>
  <action name="actionLabelMaintain">
   <property name="text">
    <string>标签维护(&amp;L)</string>
   </property>
  </action>
  <action name="actionAddLabel">
   <property name="text">
    <string>添加标签(&amp;E)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
